packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -32,6 +32,12 @@ Object {
     "bgSecondary": "#F8FAFC",
     "bgSuccess": "#ECFDF5",
     "bgWarning": "#FFFBEB",
+    "borderBorderError": "#B91C1C",
+    "borderBorderInfo": "#102EE9",
+    "borderBorderPrimary": "#6C7684",
+    "borderBorderSecondary": "#D2D7DF",
+    "borderBorderSuccess": "#047857",
+    "borderBorderWarning": "#B05C09",
```

Comment:
God works in mysterious ways. So does github actions cache. These should be changed automatically now or at least the tests should be failing
```suggestion    "borderError": "#B91C1C",
    "borderInfo": "#102EE9",
    "borderPrimary": "#6C7684",
    "borderSecondary": "#D2D7DF",
    "borderSuccess": "#047857",
    "borderWarning": "#B05C09",
```

Reply:
I actually forgot to update the snapshots the second time, then did and committed the new snapshots but now the tests are failing. Should I just re-run them or am I missing something?